appinfo/info.xml

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0"?>
 <info>
 	<id>twofactor_totp</id>
-	<name>Two Factor TOTP Provider</name>
-	<description>A Two-Factor-Auth Provider for TOTP (e.g. Google Authenticator)</description>
+	<name>2-Factor Authentication</name>
+	<summary>Increase access security with 2FA/TOTP</summary>
+	<description>2-Factor Authentication adds another layer of access security to ownCloud. The authentication method confirms a user’s identity utilizing a combination of two independent components. Instead of simply requiring a username/password combination, an additional access token (time-based one-time password/TOTP), which can be obtained via mobile phones for instance, is needed to gain access to ownCloud accounts. This not only improves access security, but also provides administrators with an option for disabling individual tokens. Time-based one-time passwords (TOTP) can be obtained through services like Google Authenticator or an Open-Source implementation of the TOTP standard like LinOTP.
+This extension provides the most simple way to add a second factor to user logins. More sophisticated 2-Factor Authentication mechanisms with additional authentication devices is available via the [PrivacyIDEA ownCloud App](https://marketplace.owncloud.com/apps/twofactor_privacyidea) or by integrating other Identity Providers that offer multi-factor authenication mechanisms.</description>
```

Comment:
Sorry, but I didn't understand. My objection is not about only community interest. In my humble opinion, advertising to another app is not related to this app's description. There can be many two-factor provider apps. Why are we suggesting specifically only one other app? If there are better implementations and we are aware of them, as a developer, I am questioning, why we are spending an effort to develop this application?

Reply:
This app is easiest to set up and to use from an admin POV.
But it has limitations (only TOTP, no sophisticated session management, not possible to roll-out second factors centrally, etc.) that other apps can tackle, e.g. you can use PrivacyIDEA with Yubikeys or other second factors and more. I just want to show people that there are alternatives if their use case is more sophisticated and can't be fulfilled by this app.

Reply:
Okay, then thanks for the explanation. I just told my concerns.